scripts/jellyfin.lua

```diff
@@ -474,6 +475,12 @@ local function align_y_change(name, data)
     set_align()
 end
 
+local function enable_overlay_on_idle()
+    if mp.get_property_bool("idle-active") and not shown then
```

Comment:
You don't need to get_property when you observe_property. Look at the other times I do this to see how.

Reply:
I tested and this function is called when idle-active turns both false and true. Because of this if you play a video, first the overlay is turned off because of this line: 

https://github.com/EmperorPenguin18/mpv-jellyfin/blob/9be821963543c165970e11fdc578609f840ba1d6/scripts/jellyfin.lua#L273

Then it turns back on again because of:

https://github.com/EmperorPenguin18/mpv-jellyfin/blob/9be821963543c165970e11fdc578609f840ba1d6/scripts/jellyfin.lua#L480

Checking if idle-active is true lets me get the correct behaviour.. Can you test it and confirm? I start mpv using this command:

```
mpv --player-operation-mode=pseudo-gui --script=scripts/jellyfin.lua --idle
```
```

Reply:
Sorry I wasn't clear. I understand that it needs to be checked if it's true. What I meant was that mpv has a mechanism to prevent reading a property twice. Functions that get called when a property changes are passed the value of the property as the second argument. If you look at align_x_change for example, I call the variable "data" and use that instead of calling get_property.

Reply:
Just pushed a commit to address this